Source/Scene/Cesium3DTileOptimizations.js

```diff
@@ -0,0 +1,125 @@
+/*global define*/
+define([
+        '../Core/Cartesian3',
+        '../Core/defaultValue',
+        '../Core/freezeObject',
+        './TileOrientedBoundingBox',
+        './TileBoundingRegion'
+    ], function(
+        Cartesian3,
+        defaultValue,
+        freezeObject,
+        TileOrientedBoundingBox,
+        TileBoundingRegion) {
+    'use strict';
+
+    /**
+     * @alias Cesium3DTileOptimizationFlag
+     *
+     * Flag denoting support for a 3D Tiles optimization
+     */
+    var Cesium3DTileOptimizationFlag = {
+        UNSUPPORTED: 0,
```

Comment:
Perhaps rename these to:
```
NOT_COMPUTED
USE_OPTIMIZATION
SKIP_OPTIMIZATION
```
In Cesium, "supported" usually refers to WebGL features.